app/src/main/java/com/example/simplemp3/service/PlayService.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.simplemp3.service
+
+import android.app.Service
+import android.content.Intent
+import android.media.MediaPlayer
+import android.os.Binder
+import android.os.IBinder
+import com.example.simplemp3.R
+
+class PlayService : Service() {
+
+    companion object {
+        const val DEFAULT_LOOP_STATE = true
+
+        private fun toTwoDigitsString(number: Int): String = if (number < 10) "0$number" else number.toString()
+
```

Comment:
Hàm này thì chỉ cần sử dụng 
```suggestion
String.format("%02d", number)
```